### PR TITLE
[DO-50] Move Firebase credential to appsettings file

### PR DESCRIPTION
**Pull request by ms-darianbenito**
Related to [DO-50](https://makingsense.atlassian.net/browse/DO-50)

Pendings:

- [x] What is secret in Firebase credential?

### CODE REVIEW

File: Doppler.Push.Api/appsettings.json [hunk 1]
```diff
@@ -16,5 +16,6 @@
     "PublicKeysFolder": "public-keys",
     "PublicKeysFilenameRegex": "\\.xml$"
   },
-  "FirebaseCredencialAdminSdk": "firebase-secret-keys/firebase-adminsdk.json"
+  "FirebaseCredential": {
+  }
```
Comment (andresmoschini) [inline Doppler.Push.Api/appsettings.json:20]: What do you think about adding here the example value?

```suggestion
  "FirebaseCredential": {
    "type": "service_account",
    "project_id": "REPLACE_WITH_FIREBASE_PROJECT_ID",
    "private_key_id": "REPLACE_WITH_PRIVATE_KEY_ID",
    "private_key": "REPLACE_WITH_PRIVATE_KEY",
    "client_email": "REPLACE_WITH_ADMINSDK_ALIAS_CLIENT_EMAIL@REPLACE_WITH_ADMINSDK_DOMAIN_SERVICE_ACCOUNT_CLIENT_EMAIL",
    "client_id": "REPLACE_WITH_CLIENT_ID",
    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
    "token_uri": "https://oauth2.googleapis.com/token",
    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
    "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/REPLACE_WITH_ADMINSDK_ALIAS_CLIENT_EMAIL%40REPLACE_WITH_ADMINSDK_DOMAIN_SERVICE_ACCOUNT_CLIENT_EMAIL"
}
```

Comment (ms-darianbenito) [reply Doppler.Push.Api/appsettings.json]: I included all values except `private_key`

Comment (andresmoschini) [reply Doppler.Push.Api/appsettings.json]: What do you think about adding it as we do with other secrets?

**Comment by rprietoms** [comment]
✔️

File: Doppler.Push.Api/appsettings.json (hunk 1 above)
Comment (ms-darianbenito) [reply Doppler.Push.Api/appsettings.json]: I decided no modify these key/values in this PR

**Pull request merged**